zb-db/src/main/java/io/zeebe/db/TransactionContext.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.db;
+
+/** Represents the shared state of a database interaction */
+public interface TransactionContext {
```

Comment:
why is that? Tbh for me it makes more sense to name it TransactionContext, since you use it for the Transaction. You can run inside a transaction it wraps internally the transaction etc.

Reply:
As the comment says it represents the shared state of database interaction. This is the only way processor is interacting with the database, right?

Reply:
Yeah the comment is wrong :sweat_smile: no it uses the column family to interact with

Reply:
Ok.